test/support/uffizzi_preview_stub_support.rb

```diff
@@ -5,34 +5,46 @@
 module UffizziPreviewStubSupport
   include ApiRoutes
 
-  def stub_uffizzi_preview_list(status, body, project_slug)
+  def stub_uffizzi_preview_list(body, project_slug)
     url = deployments_uri(Uffizzi.configuration.server, project_slug)
 
-    stub_request(:get, url).to_return(status: status, body: body.to_json)
+    stub_request(:get, url).to_return(status: 200, body: body.to_json)
   end
 
-  def stub_uffizzi_preview_delete_success(status, project_slug, deployment_id)
+  def stub_uffizzi_preview_delete_success(project_slug, deployment_id)
     url = deployment_uri(Uffizzi.configuration.server, project_slug, deployment_id)
 
-    stub_request(:delete, url).to_return(status: status)
+    stub_request(:delete, url).to_return(status: 204)
   end
 
-  def stub_uffizzi_preview_delete_failed(status, body, project_slug, deployment_id)
+  def stub_uffizzi_preview_delete_failed(body, project_slug, deployment_id)
     url = deployment_uri(Uffizzi.configuration.server, project_slug, deployment_id)
 
-    stub_request(:delete, url).to_return(status: status, body: body.to_json)
+    stub_request(:delete, url).to_return(status: 404, body: body.to_json)
+  end
+
+  def stub_uffizzi_preview_create(body, project_slug)
+    url = deployments_uri(Uffizzi.configuration.server, project_slug)
+
+    stub_request(:post, url).to_return(status: 201, body: body.to_json)
   end
 
-  def stub_uffizzi_preview_create(status, body, project_slug)
+  def stub_uffizzi_preview_create_not_found(body, project_slug)
     url = deployments_uri(Uffizzi.configuration.server, project_slug)
 
-    stub_request(:post, url).to_return(status: status, body: body.to_json)
+    stub_request(:post, url).to_return(status: 404, body: body.to_json)
+  end
+
+  def stub_uffizzi_preview_update(body, project_slug, deployment_id)
+    url = deployment_uri(Uffizzi.configuration.server, project_slug, deployment_id)
+
+    stub_request(:put, url).to_return(status: 200, body: body.to_json)
   end
 
-  def stub_uffizzi_preview_update(status, body, project_slug, deployment_id)
+  def stub_uffizzi_preview_update_not_found(body, project_slug, deployment_id)
     url = deployment_uri(Uffizzi.configuration.server, project_slug, deployment_id)
 
-    stub_request(:put, url).to_return(status: status, body: body.to_json)
+    stub_request(:put, url).to_return(status: 404, body: body.to_json)
   end
 
   def stub_uffizzi_preview_describe(status, body, project_slug, deployment_id)
```

Comment:
Why is `status` still used in this stub even though it was removed from the others?